ts/types/pagopa.ts

```diff
@@ -223,6 +225,10 @@ export type CreditCardPaymentMethod = RawCreditCardPaymentMethod &
   PaymentMethodRepresentation &
   WithAbi;
 
+export type PrivativePaymentMethod = CreditCardPaymentMethod & {
```

Comment:
I was thinking about this and maybe the cleanest solution is to declare:

```typescript
export type RawPrivativePaymentMethod = WalletV2WithoutInfo & {
  kind: "Privative";
  info: CardInfo;
};

export type PrivativePaymentMethod = RawPrivativePaymentMethod & { ... };
```
In order to have a a specific type inside the domain model. What you think?

Reply:
LGTM, added in [a62709b](https://github.com/pagopa/io-app/pull/2878/commits/a62709b0988fa675a94bbf686622446c2c2e1812)